test/entry.spec.js

```diff
@@ -214,14 +350,14 @@ Object.keys(testAPIs).forEach((IPFS) => {
         assert.strictEqual(err.message, 'Ipfs instance not defined')
       })
 
-      it('throws an error if hash is undefined', async () => {
+      it('throws an error if CID is undefined', async () => {
         let err
         try {
           await Entry.fromMultihash(ipfs)
         } catch (e) {
           err = e
         }
-        assert.strictEqual(err.message, 'Invalid hash: undefined')
+        assert.strictEqual(err.message, 'Invalid multihash: undefined')
```

Comment:
This prolly should be "Invalid CID: undefined"?

Reply:
Since the function is `fromMultihash`, I decided to keep it like so. Let me know if you want this changed.

Reply:
Since `fromMultihash` is a proxy to `fromCID` now, it's now `Invalid CID` like you said.